### ISSUE TITLE
Dota 2 S2FM tools Launch Options not working

### ISSUE THREAD

**Issue by mgn-nikey** [ValveSoftware/Dota2-Gameplay#19993]
### Description

When I try to use Launch Options from SFM, such as -sfm_resolution 2160, it doesn't affect anything.

**Comment by S1lv3r4ng3l** [comment]
Have you try something like : 
`-sfm_resolution 2160 -w 3840 -h 2160`
?

**Comment by mgn-nikey** [comment]
Yes, it doesn't affect S2FM tools. w/h parameters affecting Dota client only.
